hazelcast/src/test/java/com/hazelcast/map/BackupTest.java

```diff
@@ -77,7 +77,7 @@ public void run() throws Exception {
     public void testGracefulShutdown() throws Exception {
         int size = 50000;
         TestHazelcastInstanceFactory nodeFactory = createHazelcastInstanceFactory(4);
-        final Config config = new Config();
+        Config config = getConfig();
```

Comment:
The name 'getConfig' is confusing. Afaik you are creating a new config; so I would call this newConfig or createConfig. Then the intent is revealed from the name.